packages/@react-stately/color/src/Color.ts

```diff
@@ -52,8 +49,10 @@ export class Color {
     }
   }
 
-  toString(format: ColorFormat) {
+  toString(format: ColorFormat | 'css') {
```

Comment:
Do we need a separate `css` format here or would we always convert to either rgba or hsla when formatting to CSS?

Reply:
My reasons for adding this rather than always doing `.toString('rgb')` and implementing the missing functions for that:
- the conversion might not be perfect regarding numeric precision (= could cause flicker)
- I think some color spaces are bigger than others (Lab), so the conversion wouldn't be lossless
- doing less computational work